### ISSUE TITLE
bug: more videos button not work sometimes

### ISSUE THREAD

**Issue by Ahmey1400** [anddea/revanced-patches#516]
### Type

Cosmetic

### Tools used

RVX Manager

### Application

Yt 19.16.39

Patch:2.225.0-dev.21

### Bug description

When I click the More Videos button in fullscreen, sometimes it does not appear properly.

### Device Environment

Android 14, oneui 6.0

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Ahmey1400** [comment]
Its doesnt happen at inotia's patches and it sometimes happens. Its like i click it but it doesnt show up so i need to close and open the video again

**Comment by Ahmey1400** [comment]
Thats what happens


https://github.com/anddea/revanced-patches/assets/143447608/c8abcaf4-9182-4648-8f7c-0c96d54bf9ac

**Comment by anddea** [comment]
Are you sure it does not happen on inotia's patches, you said it doesn't happen always, maybe you have to give it a time. I cannot reproduce this issue on my side to understand why it happens. Did it ever happen before, with old patches?

**Comment by Ahmey1400** [comment]
Yes on inotia's patches it wont happen, i think it doesnt load the more videos section and thats causing this problem because if i exit from fullscreen and load that videos it shows perfectly.

**Comment by anddea** [comment]
Then use inotia, the code should be the same (as for inotia patches), but if it acts weird that's the best solution for you.

**Comment by anddea** [comment]
This option is merged with `Hide latest videos` in the latest release of patches. So no problems should occur.
